.travis.yml

```diff
@@ -1,13 +1,20 @@
 language: python
-python: 2.7
+python: 
+    - 2.6
+    - 2.7
+    - 3.3
+    - 3.4
+    - pypy
```

Comment:
It should remain 2.7 only - this Python version is used to run tox. It is TOXENV variable which allows us to set the Python version. After this change Travis will run Python 2.6 tests 5 times, Python 2.7 tests 5 times, etc.